src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/readytolist/ReadyToListAboutToSubmitHandler.java

```diff
@@ -67,10 +66,13 @@ public PreSubmitCallbackResponse<SscsCaseData> handle(CallbackType callbackType,
             return HearingHandler.GAPS.handle(sscsCaseData, hearingRequestHandler);
         }
 
-        if (SscsHelper.hasHearingScheduledInTheFuture(sscsCaseData) && warningsShouldNotBeIgnored(callback)) {
+        HearingsGetResponse hearingsGetResponse = hmcHearingApiService.getHearingsRequest(sscsCaseData.getCcdCaseId(), HmcStatus.LISTED);
```

Comment:
can we maybe extract this to a common method, given that we're doing it in multiple places